### PR TITLE
[no ticket] Setup react environment

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- Install initial reactjs app based on https://github.com/facebook/create-react-app
- Add initial folder structure
- Add one component to test folder structure and works all good
- Update Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,52 @@
...
\ No newline at end of file
+# Doppler new webapp with reactjs
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Install
+
+In the project directory, you can run:
+
+`npm install` or `yarn install`
```
Comment (andresmoschini) [inline README.md]: Why not define one of them right now? I think that it is not recommendable mixing both.

Comment (andresmoschini) [reply README.md]: BTW (I prefer Yarn)

File: src/components/Header/Nav.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+export default class HeaderNav extends Component {
+	render() {
+		return (
+			<nav>
+				<ul>
+					<li>Main</li>
+				</ul>
+			</nav>
+		);
+	}
```
Comment (andresmoschini) [inline src/components/Header/Nav.js]: tabs?

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import * as serviceWorker from './serviceWorker';
```
Comment (mvillaseco) [inline src/index.js]: What are your plans about this? (it's seems dangerous)

File: README.md (hunk 1 above)
Comment (fcoronelmakingsense) [reply README.md]: yes i use Yarn but i don't know if other prefeer use npm, but we can choose one and use only this

File: package.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    "test": "react-scripts test",
+    "eject": "react-scripts eject"
+  },
+  "eslintConfig": {
+    "extends": "react-app"
+  },
+  "browserslist": [
+    ">0.2%",
+    "not dead",
+    "not ie <= 11",
```
Comment (mvillaseco) [inline package.json]: last version of IE it's 11, why not just to clarify this subject, just say: "not ie at all"

**Comment by andresmoschini** [comment]
Hi @fcoronelmakingsense, it is great having this PR! thanks!

I will take some time reading a little more and documenting myself if you don't care.

In the meantime, what do you think about adding a _.editorconfig_ file and _prettier_ to ensure a consistent styling in the future? You can see references about how we are using prettier in:

* A recent [DataHub PR](https://github.com/MakingSense/Customer-Data-Hub/pull/298) (@joaquinicolas was working on that)
* A [self trainning](https://github.com/MakingSense/doppler-feature-toggle-js) done for me.

File: package.json (hunk 4 above)
Comment (andresmoschini) [reply package.json]: I think that we need to support IE 11, there are IE 11 in Doppler.

File: src/components/Header/Nav.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+
+import './Nav.css';
+
+export default class HeaderNav extends Component {
+	render() {
+		return (
+			<nav>
+				<ul>
+					<li>Main</li>
```
Comment (cbernat) [inline src/components/Header/Nav.js]: Are we using tabs spaces for tabulation? we were using two spaces in angular

File: package.json (hunk 4 above)
Comment (cbernat) [reply package.json]: Yes we need IE 11 sadly.

Comment (andresmoschini) [reply package.json]: No so sadly, in this project we have a transpiling, so, it is not a big issue.

File: src/index.js (hunk 3 above)
Comment (fcoronelmakingsense) [reply src/index.js]: i'dont know. I can research about this service. it's add automatically when install react

File: src/components/Header/Nav.js (hunk 5 above)
Comment (fcoronelmakingsense) [reply src/components/Header/Nav.js]: fixed

**Comment by fcoronelmakingsense** [comment]
> Hi @fcoronelmakingsense, it is great having this PR! thanks!
> 
> I will take some time reading a little more and documenting myself if you don't care.
> 
> In the meantime, what do you think about adding a _.editorconfig_ file and _prettier_ to ensure a consistent styling in the future? You can see references about how we are using prettier in:
> 
> * A recent [DataHub PR](https://github.com/MakingSense/Customer-Data-Hub/pull/298) (@joaquinicolas was working on that)
> * A [self trainning](https://github.com/MakingSense/doppler-feature-toggle-js) done for me.

Yes @andresmoschini i can check this. thanks very much!

File: src/components/Header/Nav.js (hunk 2 above)
Comment (fcoronelmakingsense) [reply src/components/Header/Nav.js]: fixed

**Comment by andresmoschini** [comment]
👍 

And _Prettier_ hehe. In the links that I have sent you there are a prettier validation for building process (and CI in the future) in that way, nobody will introduce _ugly_ code.

I did not try it, but also seems useful: https://github.com/jedmao/eclint it is to validate editorconfig conventions.

Thanks again!

File: src/index.js (hunk 3 above)
Comment (joaquinicolas) [reply src/index.js]: Service worker is for PWA.

**Pull request merged**